DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/src/kernel_args_restrict.cpp

```diff
@@ -111,6 +167,25 @@ void RunKernels(size_t size, std::vector<int> &in, std::vector<int> &nr_out,
   // 'nr_out' and 'r_out' vectors).
 }
 
+// Checks if the content of two vectors are equal up to a specified size.
+// The function prints a message indicating the index at which the mismatch occurred, 
+// and the name of the kernel of which the result is checked, specified by `kernel_name`.
+bool isEqual(std::vector<int> &in, std::vector<int> &out, std::string &&kernel_name, size_t size) {
```

Comment:
```suggestion
bool IsEqual(std::vector<int> &in, std::vector<int> &out, std::string &&kernel_name, size_t size) {
```

code sample code style is 

* Variable names should be `snake_case`
* Function, class, type names should be `PascalCase()`

Reply:
Updated.